src/responder/component.ts

```diff
@@ -232,6 +242,11 @@ export class MultiResponderComponent extends Component<ResponderAnchorState, Blo
             case ResponderActionKind.EndResponse:
                 await this.responder.endResponse(action.appointmentId);
                 break;
+            case ResponderActionKind.ResponderLowBalance:
+                if((await this.responder.signer.provider!.getBalance(this.responder.address)).gt("500000000000000000")){
```

Comment:
The value `500000000000000000` should be a configurable. Set is a constructor argument, low balance warning, document it, and then pass in the `500000000000000000` when a new responder is instantiated.